sdk/servicebus/Azure.Messaging.ServiceBus/src/Transports/TransportBody.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Messaging.ServiceBus.Transports
+{
+    internal abstract class TransportBody
```

Comment:
Suggest putting this in Azure.Messaging.ServiceBus.Core and to go with the other Transport abstractions.

Reply:
+1

Reply:
Sounds good. Will change.

Reply:
Done.